src/components/Bookcard.js

```diff
@@ -11,7 +18,12 @@ export default function Bookcard({ title, author }) {
         <p>{author}</p>
         <div className="Buttons">
           <button type="button">Comment</button>
-          <button type="button">Remove</button>
+          <button
+            type="button"
+            onClick={removeBookHandler}
```

Comment:
- [ ] Kindly ensure that your removeBookHandler function takes the id from the book data and dispatches the `removeBook` action creator with the id as its argument